src/vmm/src/devices/virtio/vhost_user_metrics.rs

```diff
@@ -140,3 +140,38 @@ pub struct VhostUserDeviceMetrics {
     // Vhost-user activate time in micro seconds.
     pub activate_time_us: SharedStoreMetric,
 }
+
+#[cfg(test)]
+pub mod tests {
+    use super::*;
+    use crate::logger::{IncMetric, StoreMetric};
+
+    #[test]
+    fn test_vhost_user_basic_metrics() {
```

Comment:
would be nice to have a brief description of what specifically we verify

Reply:
added a few lines.